src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -59,7 +59,534 @@
   </h2>
 
 <!-- <% String url = (String)request.getAttribute("url");%> -->
-<img src="${pageContext.request.contextPath}/images/foo.png" alt="Profile Image Here" width="500" height="333">
+<!-- <img src="${pageContext.request.contextPath}/images/foo.png" alt="Profile Image Here" width="500" height="333"> -->
+
+<img src="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcU
```

Comment:
Cool!, We can totally store this base64 string in the servlet. Once we done that we can do something like
<img src=<%= request.getSession().getAttribute("user").getProfilePicBytes()%> alt="Profile Image Here" width="500" height="333">
I think.

Reply:
We could do that, but would probably later want to change it so that it 'links' to the media servlet like the commented part above. Apparently it's better practice, but I guess to speed things up, this is the way to go.

Reply:
Sure, I am not asking you to do this here in this pull request.
I am simply excited to see you have proven that you can take a base64 bytes string and display the image. That is really good progress. Base64 bytes can be easily manipulated / stored by the backend.

Reply:
That's good to know. I tried storing into the class I made, but due to the '/' messing up the string, it's kind of hard to do, but I know that the media can be stored because I've seen the entities show up when I saved them. I think it would be best to figure out how to make it so that the use can store an image and them make that image appear.

Reply:
I think this is a good point to begin splitting up the work, so I'll wait for us to discuss it tomorrow if that's okay.